### PR TITLE
Feature to allow bypassing all OSBS operator processing

### PR DESCRIPTION
**Pull request by rcerven**
* CLOUDBLD-2425

Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

File: docs/admins.rst [hunk 1]
```diff
@@ -573,6 +576,11 @@ registry_post_replace
   replacing namespace/repo, all ``old`` registries in image pullspecs will be
   replaced by their ``new`` replacements.
 
+skip_all_allow_list
+  List of koji packages (with comment about The Vault Exception ID) which are
```
Comment (MartinBasti) [inline docs/admins.rst:580]: What is `Vault Exception ID` from upstream POW?

Comment (rcerven) [reply docs/admins.rst:580]: @lcarva do you know?

Comment (lcarva) [reply docs/admins.rst:580]: How about we omit `(with comment about The Vault Exception ID)` from the upstream documentation completely? This is an internal business requirement.

Comment (rcerven) [reply docs/admins.rst]: removed

**Review by ben-alkov — APPROVED** [review]

**Review by lcarva — APPROVED** [review]

**Pull request merged**